src/Base/Core/Diagnostics.cs

```diff
@@ -138,6 +123,10 @@ static Diagnostics()
         {
             // Overwrite file
             File.Create(_fileName).Close();
```

Comment:
This doesn't work for the Android build but I have no out of the box solution atm.